packages/datagrid/src/components/default-header-renderer/quality-bar.component.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import theme from './quality-bar.scss';
+
+export default function QualityBar(props) {
+	return (
+		<div className={classNames(theme['td-quality-bar'], 'td-quality-bar')}>
+			<span
+				className={classNames(theme['td-quality-bar-invalid'], 'td-quality-bar-invalid')}
+				style={{ width: `${props.invalid}%` }}
+				title={`${props.invalid}%`}
+			/>
+			<span
+				className={classNames(theme['td-quality-bar-empty'], 'td-quality-bar-empty')}
+				style={{ width: `${props.empty}%` }}
+				title={`${props.empty}%`}
+			/>
+			<span
+				className={classNames(theme['td-quality-bar-valid'], 'td-quality-bar-valid')}
+				style={{ width: `${props.valid}%` }}
+				title={`${props.valid}%`}
```

Comment:
Each bar should only be displayed if value > 0
If it is displayed, it must have a min width to be easily clickable